ontology/foodon.md

```diff
@@ -37,6 +37,27 @@ repository: https://github.com/FoodOntology/foodon
 tags:
 - food
 tracker: https://github.com/FoodOntology/foodon/issues/
+usages:
+- description: FoodData Central nutrition database web portal provided by USDA Agricultural Research Service.
+  examples:
+  - description: An entry for 'Apples, fuji, with skin, raw' from the FoodData Central nutrition database which is annotated with the term FOODON:00002862.
+    url: https://fdc.nal.usda.gov/fdc-app.html#/food-details/1750340/attributes
+  user: https://fdc.nal.usda.gov/
+- description: FDA GenomeTrackr surveillance program for reporting foodborne pathogen biosamples.
+  examples:
+  - description: An entry from NCBI Biosample that describes a *Samonella enterica* sample extracted from Chicken and annotated with the term FOODON:03411457.
+    url: https://www.ncbi.nlm.nih.gov/biosample/SAMN03455272
+  user: https://www.fda.gov/food/whole-genome-sequencing-wgs-program/genometrakr-network
+- description: Periodic Table Of Food Initiative database of food biosample nutrition analytics.
+  examples:
+  - description: <TODO>
```

Comment:
@ddooley Unfortunately I was not able to explore this particular example since it requires a login.

Reply:
Ok, lets drop PTFI then.  Its free access but login required.  Maybe later they'll make it without login.